movement/io/load_poses.py

```diff
@@ -678,3 +679,102 @@ def _ds_from_valid_data(data: ValidPosesDataset) -> xr.Dataset:
             "source_file": None,
         },
     )
+
+
+def from_mmp_file(
+    file_path: Union[Path, str], fps: Optional[float] = None
+) -> xr.Dataset:
+    """Load pose tracking data from a JSON file with multiple individuals.
+
+    This function expects JSON data in the following format:
+    [
+        {
+            "frame_id": int,
+            "instances": [
+                {
+                    "keypoints": list[list[float]],
+                    "keypoint_scores": list[float],
+                    "bbox": list[float],
+                    "bbox_score": float
+                },
+                ...
+            ]
+        },
+        ...
+    ]
+
+    Parameters
+    ----------
+    file_path : pathlib.Path or str
+        Path to the JSON file containing the pose tracking data.
+    fps : float, optional
+        The number of frames per second in the video. If None (default),
+        the 'time' coordinates will be in frame numbers.
+
+    Returns
+    -------
+    xr.Dataset
+        Dataset containing the pose tracks, confidence scores, and metadata.
+
```

Comment:
To conform to the updated phrasing of the docstrings in this module.
```suggestion
    xarray.Dataset
        ``movement`` dataset containing the pose tracks, confidence scores,
        and associated metadata.
```